src/data/pnat/basic.lean

```diff
@@ -239,6 +239,27 @@ begin
   exact hi b hk
 end
 
+/-- An induction principle for `pnat`: it takes values in `Sort*`, so it applies also to Types,
+not only to `Prop`. -/
+-- Proof by Mario Carnerio
+-- https://tinyurl.com/4m9r5yf8
```

Comment:
This belongs in the commit message not the code

Reply:
Removed!  It was also already in the commit message.